elections/code-of-conduct/2023/README.md

```diff
@@ -0,0 +1,77 @@
+# 2023 VOTERS GUIDE - KUBERNETES CODE OF CONDUCT ELECTIONS
+
+## Important Links
+
+* [Kubernetes Code of Conduct Committee]
+* [election app]
+* [election page]
+* [schedule](#schedule)
+
+## Table of Contents
+
+* [Purpose](#purpose)
+* [Election Platform](#election-platform)
+* [Eligibility](#eligibility)
+* [Schedule](#schedule)
+* [Candidacy](#candidacy-process)
+* [Voters](#voters)
+* [Nominees](#nominees)
+
+## Purpose
+
+The role of this election is to fill out the two (2) seats due for
+reelection this year on the [Kubernetes Code of Conduct Committee]. Each elected
+member will serve a two (2) year term.
+
+## Election Platform
+
+We will be using [Elekto] to conduct the election. Elekto relies entirely on GitHub Oauth for
+voting, and as such does not use email at all. Elekto also handles exceptions,
+eligibility checks, and other aspects of the election. See the [Elekto voting documentation]
+for how to use it.
+
+## Eligibility
+
+* Candidate commits to putting the interests of the community above the interests of their employer for all Code of Conduct Committee activities.
+* Is generally a responsible human
+* Does not have to be part of the Kubernetes or CNCF community
+* Characteristics which would be nice in any nominations:
+  * Previous experience on an Ethics Committee or Code of Conduct Committee is appreciated
+  * Has demonstrated integrity, professionalism, and positive influence within the community is appreciated
+  * Experience with the tools which we use to communicate (Zoom, Slack, GitHub, etc.) within the Kubernetes community is appreciated
+
+### Schedule
+
+| Date                    | Event                                             |
+|:------------------------|:--------------------------------------------------|
+| July 25                 | Nominations Open                                  |
+| Friday, August 4        | Nominations Close                                 |
+| Monday, August 7        | Steering to confirm candidacy                     |
+| Wednesday, August 9     | Election Begins                                   |
```

Comment:
Did the election already begin?

Reply:
Elekto should start the election when the PR merges. Considering the PR is still not merged, it hasn't begun.

Reply:
Understood. Should the start date be updated to 10th then (with subsequent updates to the end date)?

Reply:
Sounds good. I have updated the start date to the 11th to give a little bit more room and adjusted the timeline accordingly.